js/parse.js

```diff
@@ -376,7 +377,19 @@ function fillCharacterData(characters, numChars, fields) {
         characters[i].bubblesEquipped = [
             mapLookup(largeBubbleMap, charEquippedBubbles[0]),
             mapLookup(largeBubbleMap, charEquippedBubbles[1])
-        ]
+        ];
+
+        const fieldsPrint = JSON.parse(fields.Print.stringValue);
+        const printData = fieldsPrint.slice(5, fieldsPrint.length); // REMOVE 5 '0' ELEMENTS
+        const chunk = 14; // # OF SAMPLE SLOTS OF ONE CHARACTER
+        const relevantPrinterData = printData.slice(i * chunk, (i * chunk + chunk));
+        characters[i].printer = [
```

Comment:
I think we should structure this as characters[i].printer.selected = [...] so in the future "printer.stored" could be added very easily which stores the pinter samples that aren't selected but still saved.